spec/lib/vcr/request_ignorer_spec.rb

```diff
@@ -55,6 +55,18 @@ def request(uri)
       RequestIgnorer::LOCALHOST_ALIASES.each do |host|
         it_behaves_like "#ignore?", "http://#{host}/foo", true
       end
+
+      it 'localhost_ignored is true' do
+        expect(subject.localhost_ignored?).to eq(true)
+      end
+    end
+
+    context 'when ignore_localhost is set to false' do
+      before(:each) { subject.ignore_localhost = false }
```

Comment:
```suggestion
      before { subject.ignore_localhost = false }
```

Reply:
By using the default, we make it easier to read.

Reply:
I didn't know that. Should I correct the other in this file as well? (ex: line 52)

Reply:
Just do this one, thanks. Other PRs can polish other things.

Reply:
done!